code/style.css

```diff
@@ -1,7 +1,315 @@
+
+/* ------------------------- HEADER ------------------------- */
+header {
+  height: 15vh;
+}
+
+video {
+  object-fit: cover;
+  width: 100vw;
+  height: 80vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  margin-right: 30px;
+}
+
+header img {
+  width: 500px;
+}
+
+header nav a {
+  font-size: 12px;
+  letter-spacing: 10px;
+  margin-left: 30px;
+  padding-bottom: 5px;  
+}
+
+
+/* ------------------------- HERO ------------------------- */
+.hero {
+  text-align: center;
+  height: 65vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.hero p {
+  font-size: 18px;
+  max-width: 45vw;
+}
+
+.hero-arrow-down {
+  width: 50px;
+  height: 50px;
+  border: 1px solid #fff;
+  border-radius: 50%;
+  margin-top: 20px;
+}
+
+.hero-arrow-down:hover {
+  border: 1px solid #FF33F6;
+}
+
+.hero-arrow-down p {
+font-size: 30px;
+}
+
+
+/* ------------------------- MARQUEE ------------------------- */
+.marquee {
+  margin-top: 20px;
+}
+
+
+/* ------------------------- SECRETFORM ------------------------- */
+.secret-form {
+  width: 80vw;
+  margin: 50px auto;
+  text-align: center;
+  border: 1px solid #fff;
+  padding: 40px;
+}
+
+label {
+  display: block;
+  margin: 20px 0 5px 0;
+}
+
+textarea, input {
+  border: 0;
+  outline: none;
+  border: 1px solid #fff;
+  font-family: monospace;
+}
+
+textarea {
+  min-height: 150px;
+  width: 65vw;
+}
+
+input {
+  height: 30px;
+  width: 250px;
+}
+
+input:focus, textarea:focus {
+  border: 2px #FF33F6 solid;
+}
+
+.checkboxes-all-wrapper {
+  margin: 30px 0;
+}
+.checkbox-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.checkbox-wrapper label {
+  padding: 0;
+  margin: 0;
+}
+
+.checkbox {
+  height: auto;
+  width: auto;
+  margin-left: 10px;
+}
+
+button {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  display: block;
+  margin: 40px auto 20px auto;
+  border: 1px solid #fff;
+  color: #fff;
+  background: none;
+  outline: none;
+  cursor: pointer;
+  font-size: 18px;
+}
+
+button:hover {
+  border: 1px solid #FF33F6;
+  color: #FF33F6;
+}
+
+form a {
+border-bottom: 1px solid #fff;
+}
+
+form a:hover {
+  border-bottom: 1px solid #FF33F6;
+}
+
+/* ------------------------- FAQ ------------------------- */
+
+.accordion {
+  width:80vw;
+  max-width: 1000px;
+  margin: 30px auto;
+}
+
+.accordion-item {
+  background: #252525;
+  margin: 20px 0;
+}
+
+.accordion-item-header {
+  margin: 20px 30px 0 20px;
+  min-height: 56px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+}
+
+.accordion-item-header::after {
+  content: "\25B8";
+  position: absolute;
+  right: 0px;
+  transition: transform .3s ease-in-out;
+}
+
+.accordion-item-header.active::after {
+  transform: rotate(90deg);
+}
```

Comment:
Nice to see that you didn't give up even though ::after felt a bit confusing for you at first 💪